### PR TITLE
User rep

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

**Comment by meirwah** [comment]
pending on https://github.com/demisto/server/pull/10084

File: Misc/reputations.json [hunk 1]
```diff
@@ -121,6 +121,15 @@
...
+      "id": "userRep",
+      "version": -1,
+      "regex": "",
+      "reputationScriptName": "",
+      "details": "user",
+      "enhancementScriptNames": []
     }
-  ]
+  ],
+  "releaseNotes":"add new user reputation"
```
Comment (dorsha) [inline Misc/reputations.json]: Sentence should start with a capital letter.

Comment (aviadl) [reply Misc/reputations.json]: @meirwah 
1. Make release notes more descriptive
2. we should add a command/script to calculate the reputation, if not at least enhancement scripts

**Comment by meirwah** [comment]
@dorsha @aviadl plz review again

**Pull request merged**